apps/api/scripts/calculate-next-season-eligibility.ts

```diff
@@ -39,23 +65,22 @@ async function calculateNextSeasonEligibility() {
   const { values } = parseArgs({
     args: process.argv.slice(2),
     options: {
-      season: {
+      airdrop: {
```

Comment:
Changed this flag to "airdrop" because it makes much more sense to me. We are calculating rewards for an "airdrop" (not "season"), it happens to be that we use the previous season's activity to determine eligibility.